### PR TITLE
Fix crio service failed issue and re-enable crio.base for EL9

### PR DESCRIPTION
**Pull request by HuijingHei**
- Fix crio service failed issue
- Re-enable crio.base for EL9
- Re-enable coreos.boot-mirror for EL9

### CODE REVIEW

**Review by travier — APPROVED** [review]

**Comment by HuijingHei** [comment]
Maybe should update the URL to according to https://github.com/openshift/installer/blob/release-4.6/data/data/rhcos.json ?
```
Error: Get "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.6/46.82.202011260640-0/x86_64/meta.json": dial tcp: lookup releases-art-rhcos.svc.ci.openshift.org: no such host
```

**Comment by HuijingHei** [comment]
https://github.com/coreos/coreos-assembler/pull/3076

File: common-el9.yaml [hunk 1]
```diff
@@ -21,11 +21,7 @@ postprocess:
      ln -sr /usr/bin/conmon /usr/libexec/crio/conmon
 
      # Use crun by default
-     sed -i '/\[crio.runtime\]/a default_runtime="crun"' /etc/crio/crio.conf
-     cat >> /etc/crio/crio.conf <<EOF
-     [crio.runtime.runtimes.crun]
-     runtime_path="/usr/bin/crun"
-     EOF
+     sed -i -e '/\[crio.runtime\]/a default_runtime="crun"' -e '/\[crio.runtime.runtimes.crun\]/{n; s/runtime_path = ""/runtime_path = "\/usr\/bin\/crun"/g}' /etc/crio/crio.conf
```
Comment (haircommander) [inline common-el9.yaml:24]: we should also drop this. we should not be enabling crun by default on rhel 9 for openshift yet

**Comment by haircommander** [comment]
good find @HuijingHei I have one nit, but otherwise LGTM, thanks!

File: common-el9.yaml (hunk 1 above)
Comment (travier) [reply common-el9.yaml:24]: Thanks for the update

**Review by travier — APPROVED** [review]
/lgtm

**Pull request merged**